synthetics.go

```diff
@@ -52,12 +52,17 @@ type SyntheticsAssertion struct {
 }
 
 type SyntheticsOptions struct {
-	TickEvery          *int     `json:"tick_every,omitempty"`
-	FollowRedirects    *bool    `json:"follow_redirects,omitempty"`
-	MinFailureDuration *int     `json:"min_failure_duration,omitempty"`
-	MinLocationFailed  *int     `json:"min_location_failed,omitempty"`
-	DeviceIds          []string `json:"device_ids,omitempty"`
-	AcceptSelfSigned   *bool    `json:"accept_self_signed,omitempty"`
+	TickEvery          *int           `json:"tick_every,omitempty"`
+	FollowRedirects    *bool          `json:"follow_redirects,omitempty"`
+	MinFailureDuration *int           `json:"min_failure_duration,omitempty"`
+	MinLocationFailed  *int           `json:"min_location_failed,omitempty"`
+	DeviceIds          []string       `json:"device_ids,omitempty"`
+	AcceptSelfSigned   *bool          `json:"accept_self_signed,omitempty"`
+	MonitorOptions     MonitorOptions `json:"monitor_options,omitempty"`
```

Comment:
If `MonitorOptions` is not a pointer, bad things can happen (see https://play.golang.org/p/c3vXmQO6GeH)

```suggestion
	MonitorOptions     *MonitorOptions `json:"monitor_options,omitempty"
```

Reply:
Thank you very much. Yes, `nil` ain't good. Thank you very much.